src/compiler/crystal/interpreter/instructions.cr

```diff
@@ -1423,7 +1423,11 @@ require "./repl"
         operands:   [offset : Int32, size : Int32],
         code:       stack_move_from(stack - offset, size),
       },
-      # >>> Tuples (1)
+      tuple_copy_element: {
+        operands:   [tuple_size : Int32, old_offset : Int32, new_offset : Int32, element_size : Int32],
+        code:       (stack - tuple_size + new_offset).copy_from(stack - tuple_size + old_offset, element_size),
+      },
```

Comment:
I wonder if it wouldn't make sense to define a `copy_from` instruction that doesn't advance the stack, or maybe as an argument to the existing `copy_from`. That way we don't keep adding instructions, and maybe we end up with a more general-purpose instruction. Totally optional, though!